REQUIRE

```diff
@@ -1,3 +1,4 @@
 julia 0.6
 BinDeps
 @osx Homebrew
+VulkanCore
```

Comment:
I'd like to avoid adding this dependency if all we need is a few type aliases. I'll defer to @SimonDanisch's thoughts on this.